gradle/libs.versions.toml

```diff
@@ -5,36 +5,29 @@ spotless = "6.25.0"
 junit = "5.10.2"
 autoservice = "1.1.1"
 
-# otel SDK and API
-opentelemetrySdk = "1.35.0"
-
 # otel protocol (OTLP)
 opentelemetryProto = "1.1.0-alpha"
 
-# otel agent, those two should be updated together
+# otel agent, we rely on the '*-alpha' and get the non-alpha dependencies transitively
+# updated from upstream agent with .ci/update-upstream.sh
 opentelemetryJavaagent = "2.1.0"
-opentelemetryJavaagentAlpha = "2.2.0-alpha-SNAPSHOT"
-
-# otel semconv java generated bindings, needs to be kept in sync with version in otel agent
-opentelemetrySemconv = "1.23.1-alpha"
+opentelemetryJavaagentAlpha = "2.1.0-alpha"
 
 # otel contrib
-opentelemetryContribAlpha = "1.34.0-alpha-SNAPSHOT"
+# updated from upstream agent with .ci/update-upstream.sh
+opentelemetryContribAlpha = "1.33.0-alpha"
 
 [libraries]
 
+# For now those aren't resolved from bom for 'upstreamAgent', thus we have to keep them explicit
```

Comment:
[for reviewer] we have to keep those two as the 'upstreamAgent' does not seem to work with `platform` dependencies, thus we need to keep those for now, each of them is only used in a single place though, this is the only reason we have to keep a version on the non-alpha otherwise both could be resolved from the `-alpha` bom.

Reply:
It does actually work, you just need to enforce the `platform` for the `upstreamAgent` configuraiton aswell:
At the moment we only did it for the `implementation`-configuration.